src/components/structures/MessagePanel.js

```diff
@@ -325,6 +337,11 @@ module.exports = React.createClass({
             this.currentGhostEventId = null;
         }
 
+        this._readReceiptsByEvent = {};
```

Comment:
this could also do with a quick comment to describe the schema

Reply:
Indeed, added!